index.html

```diff
@@ -165,69 +197,78 @@ <h1>About me</h1>
+      <div class="grid2">
+        <div class="topic">
+          <img src="./svg/frameworks.svg" alt="" />
+          <p>Frameworks</p>
+        </div>
+        <ul class="discussion">
+          <li><a href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)">Bootstrap</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/Ruby_on_Rails">Ruby On Rails</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/RSpec">RSPec</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/Capybara_(software)">CapyBara</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/Selenium_(software)">Selenium</a></li>
+        </ul>
       </div>
-      <ul class="discussion">
-        <li><a href="https://en.wikipedia.org/wiki/CodeLite">CodeKit</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/GitHub">Github</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/CodePen">Codepen</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/GitLab">GitLab</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/Linux_console">Terminal</a></li>
-      </ul>
+       <div class="grid3">
+        <div class="topic">
+          <img src="./svg/skills.svg" alt="" />
+          <p>Skills</p>
+        </div>
+        <ul class="discussion">
+          <li><a href="https://en.wikipedia.org/wiki/CodeLite">CodeKit</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/GitHub">Github</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/CodePen">Codepen</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/GitLab">GitLab</a></li>
+          <li><a href="https://en.wikipedia.org/wiki/Linux_console">Terminal</a></li>
+        </ul>
+       </div>
     </div>
   </section>
   <div class="form-section">
-    <h2>
-      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-    </h2>
-    <form class="from1" action="https://formspree.io/f/xdoblzqe" method="post">
-      <ul class="inputs">
-        <li class="input1">
-          <input name="user-name" class="name" type="text" placeholder="Full Name" maxlength="30" required>
-        </li>
-        <li class="separator1">
-          <div></div>
-        </li>
-        <li class="input1 email">
-          <input  name="user-email" class="e-mail" type="email" placeholder="Email address" required>
-        </li>
-        <li class="separator1">
-          <div></div>
-        </li>
-        <li class="input1 input2"><textarea class="text-area" placeholder="Enter text here" maxlength="500" required></textarea></li>
-        <li class="form-button">
-          <button class="submit" type="submit">Get In Touch</button>
-        </li>
-      </ul>
-    </form>
+    <div class="form-grid">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
+      <form class="from1" action="https://formspree.io/f/xdoblzqe" method="post">
```

Comment:
- [ ] The form details contain approx 35% of the window width and the form contains the remaining width, Please make sure you follow the width for each section and your design should match the Figma template

### Your design

![image](https://user-images.githubusercontent.com/83803414/161121855-68b6ede4-e223-460c-af82-60b86fa35784.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161121904-83e03dfc-40d6-48b2-9522-32498515005d.png)